uds/translator/data_record/mapping_data_record.py

```
@@ -32,21 +34,25 @@ def values_mapping(self) -> MappingProxyType[int, str]:
 
         :param value: Mapping to set.
 
         :raise TypeError: Provided value is not dict type.
-        :raise ValueError: At least one key is out of raw values range.
+        :raise ValueError: If any mapping key is outside the range of valid raw values,
+            or any mapping value is not str type.
         """
-        if not isinstance(value, dict):
-            raise TypeError(f"Provided value is not dict type. Actual type: {type(value)}.")
+        if not isinstance(value, Mapping):
+            raise TypeError(f"Provided value is not Mapping. Actual type: {type(value)}.")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)